pom.xml

```diff
@@ -2055,7 +2055,7 @@
             <dependency>
                 <groupId>commons-configuration</groupId>
                 <artifactId>commons-configuration</artifactId>
-                <version>1.9</version>
+                <version>1.10</version>
```

Comment:
From maven central repo : https://mvnrepository.com/artifact/commons-configuration/commons-configuration
```
Note: This artifact was moved to: 
org.apache.commons » commons-configuration2
```
Should we try to move to it? https://mvnrepository.com/artifact/org.apache.commons/commons-configuration2/2.5

Reply:
> Should we try to move to it? https://mvnrepository.com/artifact/org.apache.commons/commons-configuration2/2.5

The short answer: we should, but it's a **very complex** task. IMO let's take the quick win then - when we have time - do the upgrade to configuration2...